napari/_qt/containers/qt_layer_list.py

```diff
@@ -44,7 +44,10 @@ class QtLayerList(QtListView[Layer]):
 
     def __init__(self, root: LayerList, parent: QWidget = None) -> None:
         super().__init__(root, parent)
-        self.setItemDelegate(LayerDelegate())
+        layer_delegate = LayerDelegate()
+        self.setItemDelegate(layer_delegate)
+        layer_delegate.loading_frame_changed.connect(self.viewport().update)
```

Comment:
Without this, I'm assuming the widget wouldn't get a render pass, so we wouldn't see movie animate? Could be worth adding an implementation comment to explain this for future readers.

Reply:
Checked this and it is mostly needed to be sure that the loading indicator is removed/not painted when appropiate and makes the animation look smoother. Also, while checking this, I noticed a missing call to pause the movie 😅 (otherwise the items will always be asking for a rerender!). Will add a comment and the pause call :+1: Thanks for pointing to check this!